.github/workflows/lxd.yaml

```
@@ -0,0 +1,47 @@
+name: LXD
+
+on:
+  push:
+    branches: [main]
+    paths-ignore: ['**.md', '**.erb', 'schemas/*']
+  pull_request:
+    type: [opened, reopened, edited]
```

```suggestion
    types: [opened, reopened, edited, synchronize]
```

Found out yesterday that we've been using the wrong key. :P